src/app/start-page/components/start-page-search/start-page-search.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="start-page-search">
```

Comment:
I'm really sorry, but I couldn't start analyzing the code yet. Maybe it's best if we do that together on Monday. The one thing that I saw when trying to get an overview was that you're using a whole lot of `@ViewChild` links which is usually not a good sign - Angular tends to have more logic directly in the code behind and less bindings to the DOM directly.
Also I saw that you're referencing `Component`s within `Directive`s - not sure if that's something one should do.

But as previously said I've not got the time yet to fully understand what's going on.